.github/workflows/docker.yml

```diff
@@ -19,11 +47,17 @@ jobs:
       with:
         username: ${{ secrets.DOCKERHUB_USERNAME }}
         password: ${{ secrets.DOCKERHUB_TOKEN }}
+    - name: Login to AWS ECR
+      uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+      with:
+          registry: ${{ secrets.AWS_ACCOUNT_ID }}.dkr.ecr.eu-central-1.amazonaws.com
```

Comment:
wouldn't it be better to use the whole URL as a parameter? something like `registry: ${{ secrets.AWS_ECR_URL }}`, that way it would be easier to change the ECR region

Reply:
Yeah, makes sense

Reply:
Great!